idaplugin/rematch/network.py

```diff
@@ -106,6 +111,8 @@ def query(method, url, server=None, token=None, params=None, json=False):
     logger('network').debug(ex.__dict__)
     if ex.code in (500,):
```

Comment:
you u can create a dict for these  error codes and return them in this way 
instead of hardcoding things and having if then else multiple times

defaultdict fn or something similar.